tests/unit/crsAdapter/MerlinAdapter.test.js

```diff
@@ -1073,5 +1073,32 @@ describe('MerlinAdapter', () => {
                 done.fail('unexpected result');
             });
         });
+
+        // fit('fetchData()', (done) => {
+        //     axios.get.and.returnValue(Promise.resolve({data: createXML(
+        //         '<ServiceBlock>' +
+        //         '<ServiceRow positionNo="1">' +
+        //         '<KindOfService>MW</KindOfService>' +
+        //         '<Service>ITL22B1/BGY-VRN1</Service>' +
+        //         '<Accommodation>0940</Accommodation>' +
+        //         '<FromDate>020218</FromDate>' +
+        //         '<EndDate>060218</EndDate>' +
+        //         '</ServiceRow>' +
+        //         '<ServiceRow positionNo="2">' +
+        //         '<KindOfService>E</KindOfService>' +
+        //         '<Service>Hotel Name</Service>' +
+        //         '<FromDate>020218</FromDate>' +
+        //         '<EndDate>060218</EndDate>' +
+        //         '</ServiceRow>' +
+        //         '</ServiceBlock>'
+        //     )}));
+        //
+        //     adapter.fetchData().then((data) => {
+        //         done();
+        //     }, (error) => {
+        //         console.log(error.message);
+        //         done.fail('unexpected result');
+        //     });
+        // });
```

Comment:
Do we want to keep commented code in source code?

Reply:
no. as you see in the comments, there are 2 todos left.
by now, this is only an intermediate state, which shall show the new structure and be open for reviews and comments